content/python/concepts/threading/terms/is-alive/is-alive.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: '.is_alive()'
+Description: 'Returns True if the thread is still running and False, otherwise.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Async Await'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.is_alive()` method returns `True` if the thread is still running and `False`, otherwise.
+
+## Syntax
+
+```pseudo
+thread_object.is_alive()
+```
+
+This method will return `True` from when the [`.run()`](https://www.codecademy.com/resources/docs/python/threading/run) method starts until just after it finishes.
+
+## Example
+
+In the example below, two `print(f'...')` statements are written to show calls to `thread.is_alive()`:
+
+```py
+import threading
+
+def countdown(count):
+  print(f'Alive? {thread.is_alive()}')
+  print("Counting down...")
+  while count > 0:
+    print(f'{count} left')
+    count -= 1
+  print("We made it!")
+
+thread = threading.Thread(target=countdown, args=(5,))
+
+thread.start()
+thread.join()
+
+print(f'Still alive? {thread.is_alive()}')
+print("End of program.")
+```
+
```

Comment:
Can we add the output for this code snippet?
😆 these examples are everything, going to refer to Frankenstein as Frankie from now on